esrally/driver/runner.py

```diff
@@ -1999,13 +2004,121 @@ def __repr__(self, *args, **kwargs):
         return "delete-transform"
 
 
+class SubmitAsyncSearch(Runner):
+    async def __call__(self, es, params):
+        request_params = params.get("request-params", {})
+        response = await es.async_search.submit(body=mandatory(params, "body", self),
+                                                index=params.get("index"),
+                                                params=request_params)
+
+        op_name = mandatory(params, "name", self)
+        # id may be None if the operation has already returned
+        search_id = response.get("id")
+        CompositeContext.put(op_name, search_id)
+
+    def __repr__(self, *args, **kwargs):
+        return "submit-async-search"
+
+
+def async_search_ids(op_names):
+    if isinstance(op_names, str):
+        subjects = [op_names]
+    else:
+        subjects = op_names
```

Comment:
or just one line `subjects = [op_names] if isinstance(op_names, str) else op_names`?

Reply:
Thanks, that looks nicer indeed. Changed in https://github.com/elastic/rally/pull/1129/commits/ed81c409ad72181111a5ffe86973ca5ec53ae8cf.